### ISSUE TITLE
Unspecified error for circular or bordered images on network nodes in IE10

### ISSUE THREAD

**Issue by mathorn** [almende/vis#3048]
When looking at the visjs.org network examples (http://visjs.org/network_examples.html) with IE10 (WIN7 virtual box from https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/), I get an 'unspecified error' on 'vis.js, line 36711, character 13'. 
This seems to be related to nodes with circular or bordered images, as I get the same error in our implementation as well as on the example pages (http://visjs.org/examples/network/nodeStyles/circularImages.html, http://visjs.org/examples/network/nodeStyles/imagesWithBorders.html).
Simple images seem to work ok (http://visjs.org/examples/network/nodeStyles/images.html),
IE11 is also working well.
Thanks for your help, best regards!

**Comment by bradh** [comment]
What is on that line? Are you possibly hitting the bad char from momentjs bug?

**Comment by wimrijnders** [comment]
When looking at `vis.js, line 36711, character 13`:

```
         } else {
            // draw image
            ctx.drawImage(this.imageObj, this.left, this.top, this.width, this.height);
          }
```
It's the 'c' in `ctx.drawImage`. I don't see anything strange here at all, even in hex mode.

**Comment by wimrijnders** [comment]
@bradh No, this is not #2912. Nothing to do with the strange character in `moment.js`.

**Comment by mathorn** [comment]
Thanks for your help @wimrijnders, that seems like a probable cause for the issue. Will wait for the fix to be accepted.

**Comment by wimrijnders** [comment]
@mathorn Since there is a fix coming up, I'll close this issue. Feel free to open it again if there is reason to do so.

**Closed. Pending fix #2964 will resolve this issue.**
